lib/Find-VS2017.cs

```diff
@@ -0,0 +1,262 @@
+// powershell -ExecutionPolicy Unrestricted -Version "2.0" -Command "&{Add-Type -Path Find-VS2017.cs; [VisualStudioConfiguration.Main]::Query()}"
```

Comment:
I wrote it myself. GUIDs extracted from [interop](https://www.nuget.org/packages/Microsoft.VisualStudio.Setup.Configuration.Interop/)